### PR TITLE
Add extra checks to id-card certificate

### PR DESCRIPTION
**Pull request by ErkoRisthein**

### CODE REVIEW

File: etc/eb/.ebextensions/nginx/conf.d/01_ssl_proxy.conf [hunk 1]
```diff
@@ -121,6 +129,7 @@ server {
 
     location /idLogin {
         expires -1;
+        # proxy_pass http://192.168.65.2:9000;
         proxy_pass http://localhost:5000;
```
Comment (v3rm0n) [inline etc/eb/.ebextensions/nginx/conf.d/01_ssl_proxy.conf:133]: Use $backend here as well.

File: etc/eb/.ebextensions/nginx/conf.d/01_ssl_proxy.conf [hunk 2]
```diff
@@ -1,3 +1,11 @@
+map $host $frontend {
+  default "http://pension.tuleva.ee.s3-website.eu-central-1.amazonaws.com";
+  # default "http://192.168.65.2:3000";
```
Comment (v3rm0n) [inline etc/eb/.ebextensions/nginx/conf.d/01_ssl_proxy.conf:3]: What is this IP? If local then use `127.0.0.1` instead. Maybe you can use this trick in the other nginx file which includes this file to have it as default https://stackoverflow.com/a/30083902

File: README.md [hunk 3]
```diff
@@ -110,6 +110,24 @@ curl --location --request POST 'https://pension.tuleva.ee/api/oauth/token' \
...
+
+In order to test ID-card locally, you need to run nginx locally with the right certificates and the right domain names.
+
+1. Add tuleva certs to `./nginx` (4 files)
+2. Update ```$frontend``` and `$backend` urls in `etc/eb/.ebextensions/nginx/conf.d/01_ssl_proxy.conf`
+3. Add to `hosts` file:
+   ```
+   127.0.0.1 id.tuleva.ee
+   127.0.0.1 pension.tuleva.ee
+   127.0.0.1 onboarding-service.tuleva.ee
```
Comment (v3rm0n) [inline README.md:123]: Should we get rid of this domain and only use pension? :thinking:

**Review by v3rm0n — APPROVED** [review]

File: etc/eb/.ebextensions/nginx/conf.d/01_ssl_proxy.conf (hunk 2 above)
Comment (ErkoRisthein) [reply etc/eb/.ebextensions/nginx/conf.d/01_ssl_proxy.conf:3]: Docker localhost

**Pull request merged**